operator/main.go

```diff
@@ -363,6 +366,13 @@ func main() {
 		setupLog.Error(err, "unable to create webhook", "webhook", "KeptnWorkloadInstance")
 		os.Exit(1)
 	}
+	if err = (&controlleroptions.KeptnConfigReconciler{
```

Comment:
I know this is autogen, but can you init and run the setup separately like we have for the others? Flo likes consistency 😃

Reply:
done